### PR TITLE
Add a WebGL 1 / 2 difference in the spec.

### PR DESCRIPTION
**Pull request by zhenyao**
ABout color conversion rules for copyTex{Sub}Image2D.

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell @jdashg @RafaelCintron PTAL

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -2630,6 +2630,22 @@ <h2 class="no-toc">Table of contents</h2>
         portion is out-of-bounds, the values may change accordingly.
     </p>
 
+    <h4>Color conversion in copyTex{Sub}Image2D</h4>
+
+    <p>
+        In WebGL 1 (OpenGL ES 2.0), it is allowed for the component sizes of
+        <em>internalformat</em> to be less than the corresponding component sizes
+        of the source buffer's internal format. However, in WebGL 2 (OpenGL ES 3.0),
+        if <em>internalformat</em> is sized, its component sizes have to exactly
```
Comment (jdashg) [inline specs/latest/2.0/index.html]: s/have to/must/

**Comment by jdashg** [comment]
lgtm

File: specs/latest/2.0/index.html (hunk 1 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

**Comment by kenrussell** [comment]
LGTM after the fact.

**Pull request merged**